.eslintrc.js

```diff
@@ -1,32 +1,40 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    ecmaVersion: 6,
-    ecmaFeatures: {
-      jsx: true,
-      experimentalObjectRestSpread: true,
-    },
-    sourceType: 'module',
-  },
-  extends: ['airbnb-base', 'plugin:react/recommended', 'prettier'],
-  plugins: ['react', 'jest'],
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
+    'prettier',
+  ],
+  plugins: ['@typescript-eslint'],
+  env: { browser: true },
   rules: {
-    'react/jsx-uses-react': 'error',
-    'react/jsx-uses-vars': 'error',
-    'import/no-extraneous-dependencies': 'off',
-    'import/no-unresolved': 'off',
-    'import/extensions': 'off',
-  },
-  env: {
-    'jest/globals': true,
-    browser: true,
+    '@typescript-eslint/no-explicit-any': 0, // We want to use any as some people are not abiding by our contract and we do not want to have Typescript throw an error quite yet
+    '@typescript-eslint/no-empty-function': 0, // We like using empty noop (for now)
+    '@typescript-eslint/explicit-module-boundary-types': 'off', // It's necessary to turn this off for non Typescript files
+    /* =========================================
+
+      We use "no-restricted-syntax" is used to
+      limit what gets rolled out in each stage.
+      https://cbinsights.atlassian.net/wiki/spaces/TD/pages/2379907163/Rollout+plan
+
+      ========================================= */
+    'no-restricted-syntax': [
```

Comment:
It seems obvious now, but just make sure this is inside `rules` and that there isn't 2 sets of `rules` or issues will occur.